src/operator-run.c

```diff
@@ -2034,23 +2041,226 @@ void xnn_compute_f16_qd8_convert(
   context->convert_ukernel(n, input, output, &params);
 }
 
+#define KAI_ASSERT(x)                                                                                                  \
```

Comment:
This seems like the wrong place to put this... it is a packing.c pack.h function.
For lhs, packx does packing on lhs for ppmm gemm microkernels.